.ci/append_comment.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#
+# [description]
+#     Update comment appending a given body to the specified original comment.
+#
+# [usage]
+#     append_comment.sh <COMMENT_ID> <BODY>
+#
+# COMMENT_ID: ID of comment that should be modified.
+#
+# BODY: Text that will be appended to the original comment body.
+
+set -e
+
+if [ -z "$GITHUB_ACTIONS" ]; then
+  echo "Must be run inside GitHub Actions CI"
+  exit -1
+fi
+
+if [ $# -ne 2 ]; then
+  echo "Usage: $0 <COMMENT_ID> <BODY>"
+  exit -1
+fi
+
+comment_id=$1
+body=$2
+
+old_comment_body=$(curl -sL \
+  -H "Accept: application/vnd.github.v3+json" \
+  -H "Authorization: token $SECRETS_WORKFLOW" \
+  "${GITHUB_API_URL}/repos/microsoft/LightGBM/issues/comments/$comment_id" | \
+  jq '.body')
```

Comment:
```suggestion
old_comment_body=$(
  curl -sL \
      -H "Accept: application/vnd.github.v3+json" \
      -H "Authorization: token $SECRETS_WORKFLOW" \
      "${GITHUB_API_URL}/repos/microsoft/LightGBM/issues/comments/$comment_id" | \
      jq '.body'
)
```

I think having the first line end with `(` and having `)` on a line by itself makes it easier to see where this begins and where it ends